typings/web3-eea/index.d.ts

```diff
@@ -1,5 +1,7 @@
-
 declare module "web3-eea" {
+    import type Web3 from "web3";
```

Comment:
@travis-payne Please create a follow-up task to remove this file and it's parent directory completely from our repo once this other PR gets merged (+released with the then latest version of web3-eea)
https://github.com/ConsenSys/web3js-eea/pull/166
(once the above happens this file will become redundant which is great because we don't want to maintain it ourselves within Cactus)